main.go

```diff
@@ -450,6 +451,20 @@ func init() {
 		}
 		log.Printf("[INFO]  : Enabled Outputs : %s\n", outputs.EnabledOutputs)
 	}
+
+	if config.Yandex.S3.Bucket != "" {
+		var err error
+		yandexS3Client, err = outputs.NewYandexS3Client(config, stats, promStats, statsdClient, dogstatsdClient)
```

Comment:
```suggestion
		yandexClient, err = outputs.NewYandexClient(config, stats, promStats, statsdClient, dogstatsdClient)
```

See my previous comment